src/frequenz/sdk/microgrid/client/_connection.py

```diff
@@ -10,7 +10,10 @@ class Connection(NamedTuple):
     """Metadata for a connection between microgrid components."""
 
     start: int
+    """The start index of the connection."""
+
     end: int
+    """The end index of the connection."""
```

Comment:
What is a start and end index of a connection? It doesn't seem very clear to me at least...

Reply:
True, I'll clarify it